### DISCUSSION TITLE
AnyDesk remote access no keyboard/mouse input in Avalonia application

### DISCUSSION THREAD

**Discussion by msneijders** [AvaloniaUI/Avalonia#13764] [Q&A] [answered]
I do not use AnyDesk (https://anydesk.com/) myself, but I got the report from some of my users that AnyDesk Remote Access does not work with an Avalonia based application. Input from keyboard/mouse does not seem to go through to the app.

Has anyone here tried AnyDesk and got it working using an avalonia app?

(ps: platform Windows10+)

**Comment by jack775544** [comment, marked as answer]
I have an Avalonia application that is deployed to a few hundred machines that all have AnyDesk on them and I have never had any issues controlling the application remotely, mouse and keyboard have always worked fine.

**Reply by msneijders** [reply]
ok, good to hear, thank you for your response -- I'll have to try it myself to see what my users are experiencing exactly
